resources/assets/less/bem/beatmap-discussion-new.less

```diff
@@ -23,21 +23,18 @@
 
   &__footer {
     display: flex;
-    flex-wrap: wrap;
+    gap: 5px;
     background-color: @osu-colour-b5;
-    padding: 5px @_side-padding;
+    padding: 10px @_side-padding;
     justify-content: space-between;
 
     @media @mobile {
-      width: 100%;
-      flex-direction: column;
```

Comment:
Button overflow due to this change

![](https://s.myconan.net/2022-08/firefox_2022-08-29_14-24-12.png)